### PR TITLE
test: add review gate golden routes

### LINKED ISSUE #280: test: review gate golden scenario tests (CHANGES_REQUESTED escalation, F2/F3 diff, awaiting-reviewer)
## Background

Current tests cover review snapshot structure (`tests/review-snapshot.test.mjs`) but lack end-to-end golden scenarios for multi-step review gate routing.

Three high-value scenarios are missing:

- **Rejected CHANGES_REQUESTED escalation**: an agent rejects reviewer feedback, the reviewer re-opens the thread, and the escalation path executes correctly
- **F2 snapshot vs F3 final fetch diff**: F3 detects new review activity that arrived after the F2 snapshot was taken and returns to E1
- **Unresolved thread + awaiting-reviewer exception**: a thread is excluded from the blocking unresolved count because its latest reply puts it in the awaiting-reviewer state (no AMD marker)

Refs #277

## Proposed change

Add golden fixture-backed scenario tests in a new `tests/review-gate.test.mjs`:

1. **CHANGES_REQUESTED escalation**: fixture with a rejection reply and a re-opened thread, verify that the E6 path routes to hold/escalation
2. **F2/F3 diff detection**: fixture with an F2 activity snapshot followed by a new review event, verify that the F3 freshness check returns to E1
3. **Awaiting-reviewer exclusion**: fixture with an unresolved thread whose latest reply is an IDD-agent reply without an AMD marker, verify it is excluded from the non-awaiting-reviewer unresolved count

## Acceptance criteria

- At least 3 golden scenario test cases exist covering the above scenarios
- Tests validate correct routing decisions (block, proceed, escalate, return to E1)
- `node --test tests/*.mjs` passes
- CI passes

## Candidate files

- `tests/review-gate.test.mjs` (new)
- `tests/fixtures/` (new fixture files)

### PR DESCRIPTION
**Pull request by kurone-kito**
## Summary
- add pure review-gate helpers for CHANGES_REQUESTED escalation, F2/F3 snapshot drift, and unresolved-thread classification
- add fixture tables for review-body timing, snapshot diff reasons, and awaiting-reviewer / AMD / conversation-resolution branches
- cover the route decisions in `tests/review-gate.test.mjs` and keep them aligned with the existing review-snapshot fixture shape

Closes #280

## Notes
- the CHANGES_REQUESTED scenarios include the still-blocking "maintainer agreed but state unchanged" branch
- the snapshot diff fixtures cover head changes, same-timestamp count growth, and CI-pass drift
- no follow-up issues identified

## Summary by CodeRabbit

* **Tests**
  * Added comprehensive fixtures and automated tests covering review-gate routing, CHANGES_REQUESTED workflows, snapshot-vs-live comparisons, and thread classification across many edge cases.

* **Chores**
  * Improved review-gate routing, snapshot-vs-live diffing, thread classification, and aggregation logic to support the new test scenarios and more robust gating behavior.

[Review Change Stack](https://app.coderabbit.ai/change-stack/kurone-kito/idd-skill/pull/289)

[Review Change Stack](https://app.coderabbit.ai/change-stack/kurone-kito/idd-skill/pull/289)

### CODE REVIEW

**Comment by kurone-kito** [comment]
_copilot-cli: review triage snapshot — IDD automation marker. Do not edit._

**Comment by kurone-kito** [comment]
@coderabbitai review

**Comment by kurone-kito** [comment]
advisory-wait: copilot-cli 16d837a458a6641c50be44d66ed8ecb6a6b4e39b 2026-05-11T08:58:49Z

**Comment by kurone-kito** [comment]
@coderabbitai review

**Comment by kurone-kito** [comment]
advisory-wait: copilot-cli 3fc493e6bc528522ef80e5f81443f277fde9fb4c 2026-05-11T09:08:29Z

File: scripts/protocol-helpers.mjs [hunk 1]
```diff
@@ -297,6 +297,182 @@ export function indexThreadsByReview(threads) {
...
+  if (hasAmd) {
+    return { classification: "amd-blocking" };
+  }
+
+  if (!(latestIsIddAgent || latestIsPrAuthor)) {
+    return { classification: "actionable-blocking" };
+  }
+
+  if (thread.reviewerReopenedAt) {
+    return { classification: "actionable-blocking" };
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs]: **Accepted** — fixed in 3fc493e: the gate now compares reopen evidence against the latest IDD/author reply time, and the fixtures cover the case where a reviewer reopened before the agent replied.

File: scripts/protocol-helpers.mjs [hunk 2]
```diff
@@ -297,6 +297,182 @@ export function indexThreadsByReview(threads) {
...
+  const iddAgentLogins = new Set(
+    (options.iddAgentLogins ?? [])
+      .map((login) => String(login ?? "").toLowerCase())
+      .filter(Boolean),
+  );
+  const prAuthorLogin = String(options.prAuthorLogin ?? "").toLowerCase();
+  const latestIsIddAgent = iddAgentLogins.has(latestAuthor);
+  const latestIsPrAuthor = Boolean(prAuthorLogin) && latestAuthor === prAuthorLogin;
+  const hasAmd = comments.some((comment) => {
+    return String(comment.body ?? "").trimStart().startsWith("**Awaiting maintainer decision**");
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs]: **Accepted** — fixed in 3fc493e: AMD classification now requires the marker text to come from an IDD-agent reply, so reviewer or author text no longer routes normal threads into the maintainer-decision path.

File: scripts/protocol-helpers.mjs [hunk 3]
```diff
@@ -297,6 +297,182 @@ export function indexThreadsByReview(threads) {
...
+  });
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs]: **Accepted** — fixed in 3fc493e: AMD detection now reuses the existing case-insensitive marker rule so casing differences do not downgrade an AMD thread to awaiting-reviewer.

File: scripts/protocol-helpers.mjs [hunk 4]
```diff
@@ -297,6 +297,182 @@ export function indexThreadsByReview(threads) {
...
+    return { classification: "amd-blocking" };
+  }
+
+  if (!(latestIsIddAgent || latestIsPrAuthor)) {
+    return { classification: "actionable-blocking" };
+  }
+
+  if (thread.reviewerReopenedAt) {
+    return { classification: "actionable-blocking" };
+  }
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs]: **Accepted** — fixed in 3fc493e: the snapshot helper now carries thread `updatedAt`, and the classifier can infer reopen-without-text evidence from that field when an explicit reopen timestamp is absent.

File: scripts/protocol-helpers.mjs [hunk 5]
```diff
@@ -297,6 +297,182 @@ export function indexThreadsByReview(threads) {
...
+
+  const comments = thread.comments?.nodes ?? [];
+  const latestComment = comments.at(-1) ?? null;
+  const latestAuthor = String(latestComment?.author?.login ?? "").toLowerCase();
+  const iddAgentLogins = new Set(
+    (options.iddAgentLogins ?? [])
+      .map((login) => String(login ?? "").toLowerCase())
+      .filter(Boolean),
+  );
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs:432]: **Accepted** — fixed in 3fc493e: truncated thread comment pages now fail safe to `actionable-blocking` instead of being treated as awaiting-reviewer or conversation-resolution eligible.

File: scripts/protocol-helpers.mjs [hunk 6]
```diff
@@ -297,6 +297,182 @@ export function indexThreadsByReview(threads) {
...
+  }
+  if (snapshotMax !== "none" && liveMax > snapshotMax) {
+    return { route: "return-to-e1", reason: "newer-activity" };
+  }
+  if (liveCount > snapshotCount) {
+    return { route: "return-to-e1", reason: "same-timestamp-count-growth" };
+  }
+
+  const snapshotCi = String(snapshot.latestCiCompletedAt ?? "none");
+  const liveCi = String(live.latestCiCompletedAt ?? "none");
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs]: **Accepted** — fixed in 3fc493e: `diffReviewSnapshot()` now compares the latest passing CI completion time (with a compatibility fallback), and the golden fixtures were updated to match the F2/F3 gate.

File: scripts/protocol-helpers.mjs [hunk 7]
```diff
@@ -297,6 +297,182 @@ export function indexThreadsByReview(threads) {
...
+  const snapshotMax = String(snapshot.maxActivityUpdatedAt ?? "none");
+  const liveMax = String(live.maxActivityUpdatedAt ?? "none");
+  const snapshotCount = Number(snapshot.totalItemCount ?? 0);
+  const liveCount = Number(live.totalItemCount ?? 0);
+  if (snapshotMax === "none" && liveCount > 0) {
+    return { route: "return-to-e1", reason: "snapshot-was-empty-now-nonempty" };
+  }
+  if (snapshotMax !== "none" && liveMax > snapshotMax) {
+    return { route: "return-to-e1", reason: "newer-activity" };
+  }
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs]: **Accepted** — fixed in 3fc493e: snapshot drift now compares activity timestamps only when both sides have real ISO values, so a `none` sentinel no longer forces a false return to E1.

File: scripts/protocol-helpers.mjs [hunk 8]
```diff
@@ -297,6 +297,182 @@ export function indexThreadsByReview(threads) {
...
+    if (result.classification === "awaiting-reviewer") {
+      summary.awaitingReviewerCount += 1;
+      continue;
+    }
+    if (result.classification === "conversation-resolve-agent") {
+      summary.conversationResolveAgentCount += 1;
+      continue;
+    }
+    if (result.classification === "conversation-resolve-author") {
+      summary.conversationResolveAuthorCount += 1;
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs:520]: **Accepted** — fixed in 3fc493e: conversation-resolution classifications now increment `actionableCount`, so those unresolved threads still block the gate until they are resolved.

File: fixtures/review-gate/snapshot-diff-routes.json [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+      "maxActivityUpdatedAt": "2026-05-11T08:00:00Z",
+      "totalItemCount": 5,
+      "latestCiCompletedAt": "2026-05-11T08:05:00Z"
+    },
+    "live": {
+      "headSha": "abc123",
+      "maxActivityUpdatedAt": "2026-05-11T08:00:00Z",
+      "totalItemCount": 5,
+      "latestCiCompletedAt": "2026-05-11T08:05:00Z"
+    },
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply fixtures/review-gate/snapshot-diff-routes.json:15]: **Accepted** — fixed in 3fc493e: the snapshot fixtures now use `latestPassingCiCompletedAt`, matching the latest-pass watermark that F2/F3 compares.

**Comment by kurone-kito** [comment]
@coderabbitai review

**Comment by kurone-kito** [comment]
advisory-wait: copilot-cli 36273252a693295e36fde584caf1007f0b0e6735 2026-05-11T12:00:55Z

File: scripts/protocol-helpers.mjs [hunk 10]
```diff
@@ -297,6 +297,215 @@ export function indexThreadsByReview(threads) {
...
+      .filter(Boolean),
+  );
+  const prAuthorLogin = String(options.prAuthorLogin ?? "").toLowerCase();
+  const latestIsIddAgent = iddAgentLogins.has(latestAuthor);
+  const latestIsPrAuthor = Boolean(prAuthorLogin) && latestAuthor === prAuthorLogin;
+  const hasAmd = comments.some((comment) => {
+    const authorLogin = String(comment.author?.login ?? "").toLowerCase();
+    return iddAgentLogins.has(authorLogin)
+      && UNSAFE_TEXT_RULES[0].pattern.test(String(comment.body ?? "").trimStart());
+  });
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs]: **Accepted** — fixed in a57a8dd, 3627325: AMD detection now uses a dedicated anchored marker constant instead of depending on the unsafe-text rule ordering.

File: scripts/protocol-helpers.mjs [hunk 11]
```diff
@@ -297,6 +297,215 @@ export function indexThreadsByReview(threads) {
...
+
+  const snapshotCi = String(
+    snapshot.latestPassingCiCompletedAt ?? snapshot.latestCiCompletedAt ?? "none",
+  );
+  const liveCi = String(
+    live.latestPassingCiCompletedAt ?? live.latestCiCompletedAt ?? "none",
+  );
+  if (snapshotCi !== liveCi) {
+    return { route: "return-to-e1", reason: "ci-pass-drift" };
+  }
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs:411]: **Accepted** — fixed in a57a8dd: snapshot drift now normalizes comparable timestamps before comparing activity and CI values, so equivalent ISO forms do not trigger false drift.

File: tests/review-gate.test.mjs [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+      fixture.name,
+    );
+
+    const expectedClassification =
+      fixture.expected.classifications[0]?.classification ?? "resolved";
+    assert.equal(
+      classifyReviewThreadForGate(fixture.threads[0], fixture.options).classification,
+      expectedClassification,
+      `${fixture.name} single-thread classification`,
+    );
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply tests/review-gate.test.mjs]: **Accepted** — fixed in a57a8dd: the single-thread assertion now runs only for single-thread fixtures, so future multi-thread tables cannot pass or fail against the wrong expectation.

File: scripts/review-activity-snapshot.mjs [hunk 13]
```diff
@@ -174,6 +174,7 @@ function fetchReviewThreads(owner, repo, prNumber) {
                 nodes {
                   id
                   isResolved
+                  updatedAt
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/review-activity-snapshot.mjs]: **Accepted** — fixed in a57a8dd: the snapshot GraphQL query no longer selects an unsupported `updatedAt` field on review threads, so `review-activity-snapshot.mjs` keeps working against the live schema.

File: scripts/protocol-helpers.mjs [hunk 14]
```diff
@@ -297,6 +297,215 @@ export function indexThreadsByReview(threads) {
...
+      .map((login) => String(login ?? "").toLowerCase())
+      .filter(Boolean),
+  );
+  const prAuthorLogin = String(options.prAuthorLogin ?? "").toLowerCase();
+  const latestIsIddAgent = iddAgentLogins.has(latestAuthor);
+  const latestIsPrAuthor = Boolean(prAuthorLogin) && latestAuthor === prAuthorLogin;
+  const hasAmd = comments.some((comment) => {
+    const authorLogin = String(comment.author?.login ?? "").toLowerCase();
+    return iddAgentLogins.has(authorLogin)
+      && UNSAFE_TEXT_RULES[0].pattern.test(String(comment.body ?? "").trimStart());
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs]: **Accepted** — fixed in a57a8dd: AMD detection is anchored to replies that start with the maintainer-decision marker, so quoted or explanatory mentions no longer trip the AMD hold path.

File: scripts/protocol-helpers.mjs [hunk 15]
```diff
@@ -297,6 +297,215 @@ export function indexThreadsByReview(threads) {
...
+  if (thread.isResolved) {
+    return { classification: "resolved" };
+  }
+  if (thread.comments?.pageInfo?.hasNextPage) {
+    return { classification: "actionable-blocking" };
+  }
+
+  const comments = thread.comments?.nodes ?? [];
+  const latestComment = comments.at(-1) ?? null;
+  const latestCommentAt = String(latestComment?.updatedAt ?? latestComment?.createdAt ?? "");
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs]: **Accepted** — fixed in a57a8dd: reopen ordering now compares the reviewer reopen time against the reply `createdAt`, so later edits do not mask a reopen that happened after the original reply.

File: scripts/protocol-helpers.mjs [hunk 16]
```diff
@@ -297,6 +297,215 @@ export function indexThreadsByReview(threads) {
...
+  const latestIsIddAgent = iddAgentLogins.has(latestAuthor);
+  const latestIsPrAuthor = Boolean(prAuthorLogin) && latestAuthor === prAuthorLogin;
+  const hasAmd = comments.some((comment) => {
+    const authorLogin = String(comment.author?.login ?? "").toLowerCase();
+    return iddAgentLogins.has(authorLogin)
+      && UNSAFE_TEXT_RULES[0].pattern.test(String(comment.body ?? "").trimStart());
+  });
+
+  if (hasAmd) {
+    return { classification: "amd-blocking" };
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs]: **Accepted** — fixed in 3627325: a thread only stays `amd-blocking` while the latest AMD reply is still awaiting an external response; a later maintainer or reviewer reply now returns it to actionable triage.

**Comment by kurone-kito** [comment]
@coderabbitai review

**Comment by kurone-kito** [comment]
advisory-wait: copilot-cli 1457806d44bbe7e469a5df11023134b84f44015c 2026-05-11T12:14:10Z

File: scripts/protocol-helpers.mjs [hunk 17]
```diff
@@ -297,6 +298,217 @@ export function indexThreadsByReview(threads) {
...
+
+  if (!(latestIsIddAgent || latestIsPrAuthor)) {
+    return { classification: "actionable-blocking" };
+  }
+
+  const reviewerReopenedAt = inferReviewerReopenedAt(thread);
+  if (!latestCommentAt && reviewerReopenedAt) {
+    return { classification: "actionable-blocking" };
+  }
+  if (reviewerReopenedAt && reviewerReopenedAt > latestCommentAt) {
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs]: **Rejected** — the live review-thread snapshot surfaces available here do not expose a reliable reopen-without-text timestamp, and the current helper intentionally treats `reviewerReopenedAt` as optional caller-supplied evidence for the pure route model. I kept the snapshot query on supported schema fields instead of inventing a synthetic signal that the GitHub APIs do not currently provide.

File: fixtures/review-gate/thread-gate-routes.json [hunk 18]
```diff
@@ -0,0 +1,509 @@
...
+  },
+  {
+    "name": "reopen before the latest IDD reply stays awaiting-reviewer",
+    "threads": [
+      {
+        "id": "THREAD-reopen-before-reply",
+        "isResolved": false,
+        "reviewerReopenedAt": "2026-05-11T08:03:00Z",
+        "updatedAt": "2026-05-11T08:05:00Z",
+        "comments": {
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply fixtures/review-gate/thread-gate-routes.json:81]: **Rejected** — same reason: these fixtures model the pure helper branch with explicit reopen evidence, but the current GitHub snapshot surfaces do not expose a portable reopen-only timestamp to populate that field in production. Leaving the snapshot normalizer schema-valid is safer than claiming coverage for an unavailable GitHub signal.

File: scripts/protocol-helpers.mjs (hunk 17 above)
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs]: **Accepted** — fixed in 1457806: reviewer reopen evidence is now normalized and applied before the AMD hold check, so a reopen-only reviewer action can override a pending maintainer-decision wait.

File: scripts/protocol-helpers.mjs [hunk 19]
```diff
@@ -297,6 +298,217 @@ export function indexThreadsByReview(threads) {
...
+    return {
+      route: "return-to-e1",
+      reason: "reviewer disagreed with the rejection and the feedback must return to triage",
+    };
+  }
+  if (reviewerDisposition === "agreed-state-cleared") {
+    return {
+      route: "proceed",
+      reason: "reviewer agreed with the rejection and cleared the changes-requested state",
+    };
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs:318]: **Accepted** — fixed in 1457806: `routeRejectedChangesRequestedReview()` no longer lets `reviewerDisposition: agreed-state-cleared` bypass an actual `CHANGES_REQUESTED` review state.

File: scripts/protocol-helpers.mjs [hunk 20]
```diff
@@ -297,6 +298,217 @@ export function indexThreadsByReview(threads) {
...
+
+  const snapshotMax = normalizeComparableTimestamp(snapshot.maxActivityUpdatedAt);
+  const liveMax = normalizeComparableTimestamp(live.maxActivityUpdatedAt);
+  const snapshotCount = Number(snapshot.totalItemCount ?? 0);
+  const liveCount = Number(live.totalItemCount ?? 0);
+  if (snapshotMax === "none" && liveCount > 0) {
+    return { route: "return-to-e1", reason: "snapshot-was-empty-now-nonempty" };
+  }
+  if (snapshotMax && liveMax && snapshotMax !== "none" && liveMax !== "none" && liveMax > snapshotMax) {
+    return { route: "return-to-e1", reason: "newer-activity" };
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs]: **Accepted** — fixed in 1457806: `diffReviewSnapshot()` now fails closed when live activity exists but the live max-activity timestamp is missing or invalid.

File: scripts/protocol-helpers.mjs [hunk 21]
```diff
@@ -297,6 +298,217 @@ export function indexThreadsByReview(threads) {
...
+  }
+  if (elapsedMs < 48 * 60 * 60 * 1000) {
+    return {
+      route: "escalate-maintainer",
+      reason: "the changes-requested review is still blocking after 24 hours with no reviewer response",
+    };
+  }
+  return {
+    route: "label-and-release",
+    reason: "the changes-requested review is still blocking after 48 hours with no escalation response",
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs:370]: **Accepted** — fixed in 1457806: the 48-hour release path now requires maintainer-escalation evidence first, and waits if the escalation is still within its own response window.

File: scripts/protocol-helpers.mjs (hunk 20 above)
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs]: **Accepted** — fixed in 1457806: snapshot freshness now compares timestamps by parsed time value, so fractional-second activity can no longer sort before a whole-second watermark.

File: scripts/protocol-helpers.mjs [hunk 22]
```diff
@@ -297,6 +298,217 @@ export function indexThreadsByReview(threads) {
...
+      summary.conversationResolveAuthorCount += 1;
+    }
+  }
+
+  return summary;
+}
+
+function inferReviewerReopenedAt(thread) {
+  const explicit = String(thread.reviewerReopenedAt ?? "");
+  if (isValidIsoTimestamp(explicit)) {
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs:529]: **Rejected** — the current GitHub review-thread snapshot surfaces available to this workflow do not expose a reliable reopen-without-text timestamp. The helper keeps `reviewerReopenedAt` as explicit caller-supplied evidence for the pure route model, but the production snapshot path cannot safely synthesize that signal from unsupported or missing API fields.

[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs:529]: **Rejected** — same reason: the snapshot normalizer intentionally stays on schema-supported fields, and there is no portable GitHub event/timestamp here that would let us populate `reviewerReopenedAt` without inventing evidence.

**Comment by kurone-kito** [comment]
@coderabbitai review

**Comment by kurone-kito** [comment]
advisory-wait: copilot-cli d3072e1589f08b44de0366f17506b0270fd0828b 2026-05-11T12:27:57Z

File: scripts/protocol-helpers.mjs [hunk 23]
```diff
@@ -297,6 +298,240 @@ export function indexThreadsByReview(threads) {
...
+    return explicit;
+  }
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply scripts/protocol-helpers.mjs:531]: **Rejected** — the same GitHub API constraint still applies here: the snapshot path does not expose a portable reopen-without-text timestamp, so I kept `reviewerReopenedAt` as explicit caller-supplied evidence for the pure route helper instead of synthesizing unsupported data.

File: fixtures/review-gate/snapshot-diff-routes.json [hunk 24]
```diff
@@ -0,0 +1,192 @@
...
+    "live": {
+      "headSha": "abc123",
+      "maxActivityUpdatedAt": "none",
+      "totalItemCount": 5,
+      "latestPassingCiCompletedAt": "2026-05-11T08:05:00Z"
+    },
+    "expected": {
+      "route": "return-to-e1",
+      "reason": "missing-live-activity-evidence"
+    }
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply fixtures/review-gate/snapshot-diff-routes.json]: **Accepted** — fixed in d3072e1: the snapshot fixture name now matches its `return-to-e1` expectation.

File: fixtures/review-gate/thread-gate-routes.json [hunk 25]
```diff
@@ -0,0 +1,545 @@
...
+      "amdBlockingCount": 0,
+      "conversationResolveAgentCount": 0,
+      "conversationResolveAuthorCount": 0,
+      "classifications": [
+        {
+          "id": "THREAD-quoted-amd",
+          "classification": "awaiting-reviewer"
+        }
+      ]
+    }
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply fixtures/review-gate/thread-gate-routes.json]: **Accepted** — fixed in d3072e1: the quoted-AMD fixture name now matches its `awaiting-reviewer` expectation.

**Comment by kurone-kito** [comment]
**Accepted** — CodeRabbit re-review was requested for `d3072e1`, and the latest CodeRabbit check is already green for that head.

**Comment by kurone-kito** [comment]
_copilot-cli: review triage snapshot — IDD automation marker. Do not edit._

**Comment by kurone-kito** [comment]
_copilot-cli: review triage snapshot — IDD automation marker. Do not edit._
